src/transformers/modeling_bart.py

```diff
@@ -371,7 +367,7 @@ def forward(
             layer_state=layer_state,  # adds keys to layer state
             key_padding_mask=decoder_padding_mask,
             attn_mask=causal_mask,
-            need_weights=self.output_attentions,
+            need_weights=output_attentions,
```

Comment:
any specific reason to call this argument `need_weights`? Could we maybe change it also to `output_attentions` for consistency?

Reply:
It was called `need_weights` earlier also and the only reason I didn't touch it was that I was not aware of any downstream impact 😅 , I'll be fixing it now

Reply:
True! Thanks for renaming it :-) We should have a consistent name across models